includes/Modules/Analytics/Advanced_Tracking/AMP_Config_Injector.php

```diff
@@ -37,11 +37,9 @@ public function inject_event_configurations( $gtag_amp_opt, $events ) {
 			$gtag_amp_opt['triggers'] = array();
 		}
 
-		foreach ( $events as $event ) {
+		foreach ( $events as $amp_trigger_key => $event ) {
```

Comment:
Strictly saying, we should maybe call this `$identifier` or `$key`, it' not specific for AMP. But not a big deal.